knimin/templates/auth_sitebase.html

```diff
@@ -49,12 +57,21 @@
             {% end %}
         </ul>
 
+        <h3>Plate Mapper</h3>
+        <ul class="mainmenu">
+            {% if admin or db.has_access(current_user, ['Admin']) %}
+            <li><a href="/pm_plate_list/">View Plate List</a></li>
+            <li><a href="/pm_plate_map/?target=sample&id=0">Create New Plate</a></li>
```

Comment:
Minor, but wouldn't it make more sense that if no parameter is passed then `/pm_plate/map/` creates a new plate, but if a parameter is passed then shows that one?